en/admin/config.rst

```diff
@@ -2301,7 +2301,7 @@ The following are other parameters. The type and value range for each parameter
 
 **vacuum_ovfp_check_duration**
 
- **vacuum_ovfp_check_duration** sets the period to keep the information collected by the **vacuum_ovfp_check_threshold**. Information that has not been updated within the setting period is deleted. If the unit of the input value is omitted, it is millisecond, and the default value is 744 hours, which corresponds to 31 days. When setting, you can use ms/s/min/h as the unit.
+ **vacuum_ovfp_check_duration** sets the period to keep the information collected by the **vacuum_ovfp_check_threshold**. Information that has not been updated within the setting period is deleted. The unit of the input value is minutes.
```

Comment:
```suggestion
**vacuum_ovfp_check_duration** specifies the duration for which data related to the count of index overflow pages, gathered by vacuum threads, is retained. Data that remains unchanged within the specified duration will be automatically removed. The value of **vacuum_ovfp_check_duration** is measured in minutes.
```

**vacuum_ovfp_check_threshold** is not responsible for collecting data related to index overflow pages, and it also doesn't impact the data collection process. I believe the collection of such data is carried out by vacuum threads based on the value defined by vacuum_ovfp_check_threshold. Instead of specifying the **vacuum_ovfp_check_threshold**, I recommend providing a detailed description of the data collection process performed by the vacuum threads and referring to that description for better clarity.

Reply:
I'm sorry for not having written this review in the previous PR.

Reply:
Thank you for your careful review.